resqpy/time_series/time_series_any_time_series.py

```diff
@@ -0,0 +1,233 @@
+"""Any time series"""
```

Comment:
As with the other class modules, I'd prefer to drop the inherited class from the filename, so: _time_series.py (which clearly requires the module containing functions to be named something else!)